src/util.cc

```diff
@@ -577,4 +588,28 @@ Local<String> UnionBytes::ToStringChecked(Isolate* isolate) const {
   }
 }
 
+#ifndef DISABLE_SINGLE_EXECUTABLE_APPLICATION
+bool IsSingleExecutable() {
+  return postject_has_resource();
+}
+
+const char* FindSingleExecutableCode(size_t* size) {
+  if (single_executable_application_code_loaded == false) {
+    postject_options options;
+    postject_options_init(&options);
+    options.macho_segment_name = "NODE_JS";
```

Comment:
Should we not wrap this into a pre-processor for macOS?

Reply:
@tony-go done!